### PR TITLE
7903432: Feature Tests - Adding five JavaTest GUI legacy automated test scripts

### PR DESCRIPTION
**Pull request by gollayadav**
Adding below automated legacy JavaTest GUI feature Test Scripts to the Jemmy regression suite and tested locally on three platforms(Linux, Windows, Mac OS) and working fine. Also verified with JDK8 on macOS.

1.Markers14.java
2.Markers15.java
3.Markers16.java
4.Markers17.java
5.Markers18.java

---------
### Progress
- [x] Change must not contain extraneous whitespace

### Issue
 * [CODETOOLS-7903432](https://bugs.openjdk.org/browse/CODETOOLS-7903432): Feature Tests - Adding five JavaTest GUI legacy automated test scripts

### Reviewers
 * [Dmitry Bessonov](https://openjdk.org/census#dbessono) (@dbessono - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jtharness pull/44/head:pull/44` \
`$ git checkout pull/44`

Update a local copy of the PR: \
`$ git checkout pull/44` \
`$ git pull https://git.openjdk.org/jtharness pull/44/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 44`

View PR using the GUI difftool: \
`$ git pr show -t 44`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jtharness/pull/44.diff">https://git.openjdk.org/jtharness/pull/44.diff</a>

</details>

### CODE REVIEW

**Comment by dbessono** [comment]
Please correct all class-level javadoc comments.

File: gui-tests/src/gui/src/jthtest/Markers/Markers14.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        mainFrame = new JTFrame(true);
+
+        mainFrame.openDefaultTestSuite();
+        addUsedFile(mainFrame.createWorkDirectoryInTemp());
+        Configuration configuration = mainFrame.getConfiguration();
+        configuration.load(CONFIG_NAME, true);
+
+        ConfigDialog config = configuration.openByKey();
+
+        int[] indexes = new int[] { 4, 5, 6, 8, 9 };
```
Comment (dbessono) [inline gui-tests/src/gui/src/jthtest/Markers/Markers14.java:52]: Is this kept here intentionally? Seems not used.

**Review by dbessono — CHANGES_REQUESTED** [review]
Please correct all class-level javadoc comments.

File: gui-tests/src/gui/src/jthtest/Markers/Markers14.java (hunk 1 above)
Comment (gollayadav) [reply gui-tests/src/gui/src/jthtest/Markers/Markers14.java]: @dbessono Thank you for reviewing. I have updated the test script now. Could you please check.

**Comment by gollayadav** [comment]
> Please correct all class-level javadoc comments.

I have corrected the javadoc comments in all the test scripts. Could you please look into it.

**Review by dbessono — APPROVED** [review]
Thank you for the updates. Look good

**Pull request closed without merging**